index.js

```diff
@@ -54,7 +70,14 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+  let avgRating = 0;
+
+  for(movie of movies){
+    avgRating = avgRating + Number(movie.imdbRating) / movies.length;
```

Comment:
We got lucky here with oreder of operations, but `Number(movie.imdbRating) / movies.length` should reallt be wrapped in parenthesis`()`